update-delete-ignore-not-found/script.ts

```diff
@@ -0,0 +1,76 @@
+import { Prisma, PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient().$extends({
+  name: "UpdateAndDeleteIgnoreNotFound",
+  model: {
+    $allModels: {
+      async updateIgnoreNotFound<T, A>(
+        this: T,
+        args: Prisma.Exact<A, Prisma.Args<T, "update">>
+      ): Promise<Prisma.Result<T, A, "update"> | null> {
+        try {
+          const context = Prisma.getExtensionContext(this) as any;
+          return await context.update(args);
+        } catch (err) {
+          if (
+            err instanceof Prisma.PrismaClientKnownRequestError &&
+            err.code === "P2025"
+          ) {
+            return null;
+          }
+          throw err;
+        }
+      },
+      async deleteIgnoreNotFound<T, A>(
```

Comment:
```suggestion
      async updateIgnoreOnNotFound<T, A>(
        this: T,
        args: Prisma.Exact<A, Prisma.Args<T, "update">>
      ): Promise<Prisma.Result<T, A, "update"> | null> {
        try {
          const context = Prisma.getExtensionContext(this) as any;
          return await context.update(args);
        } catch (err) {
          if (
            err instanceof Prisma.PrismaClientKnownRequestError &&
            err.code === "P2025"
          ) {
            return null;
          }
          throw err;
        }
      },
      async deleteIgnoreOnNotFound<T, A>(
```

Just being a little pedantic with function naming. This would be completely up to you